src/engine.jl

```diff
@@ -197,7 +197,7 @@ function make_getvar_statement(ex::Expr)
     :( $(v) = MATLAB.get_variable($(Meta.quot(v)), $(k)) )
 end
 
-function _mget_multi(vs::Union(Symbol, Expr)...)
+function _mget_multi(vs::Union{Symbol, Expr}...)
```

Comment:
You'll need to add `@compat` in front of this line if you want this package to still be usable for julia 0.3. Alternatively, we could bump the julia version in the REQUIRE file. Personally I think the latter is fine, if you think that's easier.